dev-docs/bidder-adaptor.md

```diff
@@ -231,12 +233,39 @@ Sample array entry for `validBidRequests[]`:
 }]
 {% endhighlight %}
 
+#### bidRequest Parameters
+
 Notes on parameters in the bidRequest object:
 - **Bid ID** is unique across ad units and bidders.
 - **Auction ID** is unique per call to `requestBids()`, but is the same across ad units.
 - **Transaction ID** is unique for each ad unit with a call to `requestBids`, but same across bidders. This is the ID that DSPs need to recognize the same impression coming in from different supply sources.
 - **Bid Request Count** is the number of times `requestBids` has been called for this ad unit.
 
+#### Referrers
+
+Referrer information is included on the `bidderRequest.refererInfo` property. This property contains the following parameters:
+
+- `referer`: a string containing the detected top-level URL.
+- `reachedTop`: a boolean specifying whether Prebid was able to walk up to the top window.
+- `numIframes`: the number of iFrames.
+- `stack`: a string of comma-separated URLs of all origins.
+
+Your integration of referrer information would look something like this:
```

Comment:
We can add something here like: 
refererInfo returns raw url. You may want to encode urls before adding it to request payload.

Reply:
removed